exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java

```diff
@@ -61,12 +63,14 @@
 public class WorkManager implements AutoCloseable {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(WorkManager.class);
 
+  public static final long EXIT_TIMEOUT = 5_000L;
```

Comment:
Three nits:

1. We'll never wait decades, so this need not be a long.
2. Please encode units in the name: `EXIT_TIMEOUT_MS`
3. The underscore is great, but not entirely needed for four-digit numbers.